src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -865,7 +865,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
         |${preambleHeader format lineRep.readName}
         |${envLines mkString ("  ", ";\n  ", ";\n")}
         |$importsPreamble
-        |${indentCode(toCompute)}""".stripMargin
+        |%s""".stripMargin.format(indentCode(toCompute))
```

Comment:
I'll roll that investigation into my PR (coming once the currently in-flight repl PRs are merged).